core/java/com/google/instrumentation/stats/Aggregation.java

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright 2016, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.stats;
+
+import com.google.instrumentation.common.Duration;
+import com.google.instrumentation.common.Function;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * An {@link Aggregation} is a series of individual measurements.
+ *
+ * <p> Each measurments can be one of either:
+ * <ul>
+ * <li> {@link DistributionAggregation}
+ * <li> {@link IntervalAggregation}
+ * </ul>
```

Comment:
I think this comment could be clearer, maybe something like:

```
+/**
 + * An {@link Aggregation} summarizes a series of individual measurements.
 + *
 + * <p> It can be one of either:
 + * <ul>
 + * <li> {@link DistributionAggregation}
 + * <li> {@link IntervalAggregation}
 + * </ul>
```